src/store/modules/records/getters.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  filteredIncomingTransfers: (state) => {
```

Comment:
Shouldn't there be an actual condition in this filter?

```javascript
filteredIncomingTransfers: (state) => {
  return state.lists.incomingTransfers.filter((record) => {
    return !record.isIgnored
  })
},
```

Reply:
Oh, oops. I was screwing around with this and forgot to fix it